File: SR/20190411/1/02+.cpp
#include <iostream>
using namespace std;

int main() {
	string dna1, dna2;
	while(cin >> dna1 >> dna2) {
		int c = 0;
		for(int i = 0; i < dna1.length(); i++) {
			if(dna1[i] == 'A' && dna2[i] != 'T') {
				c = 1;
				continue;
			} else if(dna1[i] == 'T' && dna2[i] != 'A') {
				c = 1;
				continue;
			} else if(dna1[i] == 'C' && dna2[i] != 'G') {
				c = 1;
				continue;
			} else if(dna1[i] == 'G' && dna2[i] != 'C'){
				c = 1;
				continue;
			}
		}
		if(c) {
			cout << "X MEN" << endl;
		} else {
			cout << "Human" << endl;
		}
	}
}

File: SR/20200604/2/02+.py
import sys

while(1):
    dna1 = input()
    
    if dna1 == "exit":
        sys.exit()
        
    dna2 = input()
    
    X = 0
    
    for i in range(max(len(dna1), len(dna2))):
        if (dna1[i] == "A" and dna2[i] != "T") or (dna1[i] == "T" and dna2[i] != "A") or (dna1[i] == "C" and dna2[i] != "G") or (dna1[i] == "G" and dna2[i] != "C"):
            X = 1
            break
    if X:
        print("X MEN")
    else:
        print("Human")